telethon/tl/custom/message.py

```diff
@@ -824,15 +826,21 @@ async def click(self, i=None, j=None,
                 This is equivalent to ``message.buttons[0][1].click()``.
 
             text (`str` | `callable`):
-                Clicks the first button with the text "text". This may
+                Clicks the first button or poll option with the text "text". This may
                 also be a callable, like a ``re.compile(...).match``,
                 and the text will be passed to it.
 
+                Please note that the first option for which the callable
```

Comment:
The first line describing this parameter already says it clicks the first option, no need to reiterate here. I think it's better to reword this as indicating the user how to select multiple options instead:

```python
If you need to select multiple options in a poll,
pass a list of indices to the ``i`` parameter.
```